package.json

```diff
@@ -18,7 +18,8 @@
 		"changeset": "npx changeset",
 		"clean": "pnpm recursive run clean && rm -rf ./.env ./node_modules",
 		"----- CI ---- used to test the codebase on every commit": "",
-		"ci": "pnpm format && pnpm lint && pnpm build && pnpm test",
+		"ci": "pnpm install && pnpm check-all",
+		"check-all": "pnpm format && pnpm lint && pnpm build && pnpm test",
```

Comment:
i need to install so that pnpm fetchPublicEnv works before running all the checks that depend on the environment variables for external contributors 

![Screenshot 2023-11-29 at 12 42 37](https://github.com/inlang/monorepo/assets/110794494/c7677610-fd32-4dab-b8a2-a5c3fb87c0b1)